workshops/views.py

```diff
@@ -1112,6 +1112,9 @@ class EventCreate(LoginRequiredMixin, PermissionRequiredMixin,
     form_class = EventForm
     template_name = 'workshops/event_create_form.html'
 
+    def get_initial(self):
+        return {'invoice_status': 'not-invoiced'}
+
```

Comment:
Why not change the default value in model? (Except that it triggers a migration.)